src/left.jl

```diff
@@ -157,4 +157,81 @@ for (op, LType, VType) in ((:adjoint, :AdjointOperator, :AbstractAdjointVecOrMat
         u
     end
 end
+
+
+# For AdjointOperator
+# Out-of-place: v is action vector, u is update vector
+function (L::AdjointOperator)(v::AbstractVecOrMat, u, p, t; kwargs...)
+    # Adjoint operator applied to v means L.L' * v
+    # For matrices: (A')v = (v'A)'
+    # This means we need to compute L.L(v', u, p, t)' 
+    # Reshape v to match the adjoint operator's expected size
+    adjv = reshape(v', size(L.L, 1), :)
+    result = L.L(adjv, u, p, t; kwargs...)
+    return reshape(result', size(L, 1), :)
```

Comment:
I don't think you need to resize if you transpose?

Reply:
reshape should not be necessary. Just update_coefficients with u and matmul with v.